functions/index.js

```diff
@@ -14,6 +14,8 @@ const _ = require('lodash');
 const functions = require('firebase-functions');
 let functionsConfig;
 let runOnFunctionFireBaseServer = false;
+let versionPackage = require('./package.json');
```

Comment:
name `versionPackage` is a too specific name for `package.json`

Reply:
Should I rename it to package?

Reply:
`package` is reserved word. So it should be `packageJSON` or something similar.

Reply:
Ok sure.